volatility3/framework/symbols/windows/pdbconv.py

```diff
@@ -498,8 +514,12 @@ def read_symbol_stream(self):
                 if self._omap_mapping:
                     address = self.omap_lookup(address)
                 stripped_name = self.name_strip(name)
+                if stripped_name in self.symbols:
+                    vollog.warning(f"{stripped_name} already present at symbols, non stripped name {name} will be used!")
+                    stripped_name = name
```

Comment:
This is going to confuse people, we need to be consistent, so I'd issue the warning but then overwrite it.  Having people not know whether to lookup by stripped name or not will just cause chaos.

Reply:
Awesome, thanks!  5:)